pyiron/atomistics/job/atomistic.py

```diff
@@ -654,6 +676,8 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
             snapshot.cell = self.output.cells[iteration_step]
         snapshot.positions = self.output.positions[iteration_step]
         indices = self.output.indices
+        if len(indices) == 0:
+            indices = self['output/generic/indices']
```

Comment:
I am a bit confused about this construction, would not it be easier to update `self.output.indices` to fall back to `self['output/generic/indices']` if it exists?

Reply:
Agreed. I had previously been unable to figure out how to do that, but it was in the `pyiron/job/interactive.GenericeInteractiveOutput.indices` property, which needed `prop=GenericOutput.indices.fget` as an argument to `_lst_from_property`.